### ISSUE TITLE
Add Git LFS usage to documentation

### ISSUE THREAD

**Issue by leewujung** [OSOceanAcoustics/echopype#133]
We now use git LFS to store large testing data files (particularly those from EK80!), so it will be good to specify that in the documentation.

Another thing to mention here (using a `.. note::` block) is that for people who have previously forked echopype before the git-lfs migration, they will need to delete the original forked repository and fork a new one. Otherwise they will end up with 2x duplicated commits since the history has been re-written during the migration!

So in summary we need:
- [x] basic pointer on how to install git LFS 
- [x] when to invoke `git lfs fetch` to get test data sets
- [x] note on the need for deleting and re-forking the repo if the repo was forked before the migration

**Comment by leewujung** [comment]
Resolved by #138 .
